osf/migrations/0014_add_colon_delim_to_s3_folders.py

```diff
@@ -0,0 +1,13 @@
+from django.db import migrations
+from addons.s3.utils import update_folder_names, reverse_update_folder_names
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('osf', '0013_institution_support_email'),
+    ]
+
+    operations = [
+        migrations.RunPython(update_folder_names, reverse_code=reverse_update_folder_names),
```

Comment:
fyi @mfraezz for suggestion/advice

I see the convenience of auto migration, but I prefer such "data migration" (comparing to model migration) to be a command that is run manually.